docs/getting_started/intro/minimize.rst

```diff
@@ -41,6 +41,14 @@ The choice of which parameters of your model should be floating in the fit can a
 
 .. code-block:: pycon
 
+    >>> # In the case of a Gaussian (e.g.)
```

Comment:
Also here there has been doubling of text.

Reply:
indeed, weird...